pkg/types/validation/installconfig.go

```diff
@@ -675,20 +683,28 @@ func validateAPIAndIngressVIPs(vips vips, fieldNames vipFields, vipIsRequired, r
 			allErrs = append(allErrs, field.Required(fldPath.Child(fieldNames.APIVIPs), "must specify VIP for API, when VIP for ingress is set"))
 		}
 
-		if len(vips.Ingress) == 1 {
```

Comment:
Huh. Was this a bug? If there's only one VIP we could never have had both v4 and v6 so the checks below would always fail.

Reply:
Yeah, looks like it